docs/quick-start/accounts.md

```diff
@@ -0,0 +1,435 @@
+---
+sidebar_position: 2
+title: Accounts
+description: Learn how to create and manage Miden accounts programmatically using Rust and TypeScript.
+---
+
+import { CodeTabs } from '@site/src/components';
+
+# Accounts
+
+Miden's account model is fundamentally different from traditional blockchains. Let's explore how to create and manage accounts programmatically.
+
+## Understanding Miden Accounts
+
+Before diving into account creation, it's essential to understand what makes Miden accounts unique compared to traditional blockchain addresses:
+
+**What Makes Miden Accounts Special:**
+
+- **Smart Contract Wallets**: Every account is a programmable smart contract that can hold assets and execute custom logic
+- **Modular Design**: Accounts are composed of reusable components (authentication, wallet functionality, etc.)
+- **Privacy Levels**: Choose between public or private storage modes
+
+Miden accounts differ from traditional blockchain addresses in fundamental ways:
+
+**Account Architecture:**
+
+- Every account is a **smart contract** with programmable logic
+- Accounts can store **assets** (fungible and non-fungible tokens) in their vault
+- Each account has **storage slots** for custom data (up to 255 slots)
```

Comment:
Suggestion: Instead of saying "up to 255 slots", maybe for these storage slot explanations, it'd be better to give an example to shape the mental model of readers, e.g. (`public_key` -> `0xabc...`).

(This assumes named storage slots which are not yet in 0.12, but since you already describe them as key-value storage I assume that's intended).